app/uk/gov/hmrc/apisubscriptionfields/logging/ApiLogger.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apisubscriptionfields.logging
+
+import javax.inject.Singleton
+
+import com.google.inject.Inject
+import uk.gov.hmrc.play.config.inject.ServicesConfig
+
+@Singleton
+class ApiLogger @Inject()(serviceConfig: ServicesConfig) {
+
+  private lazy val loggerName: String = serviceConfig.getString("appName")
+  lazy val logger = play.api.Logger(loggerName)
+
+  def debug(s: String): Unit = logger.debug(s)
```

Comment:
If this gets merged, we should pass the parameters by name, not by value.
Use `call-by-name` way (`x: => type`)
https://stackoverflow.com/questions/13337338/call-by-name-vs-call-by-value-in-scala-clarification-needed

Reply:
not needed as deleted